recipes/upstream/google-cloud/gke/first_setup.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Log into the GKE cluster with the credentials used to create it.
+export KUBECONFIG="./$(cat ./terraform.tfvars | grep -v "#" | grep -i prefix | awk -F= '{print $2}' | tr -d '"' | sed 's/ //g')_kube_config.yml"
+gcloud container clusters get-credentials $(cat ./terraform.tfvars | grep -v "#" | grep -i prefix | awk -F= '{print $2}' | tr -d '"')-cluster --region $(cat ./terraform.tfvars | grep -v "#" | grep -i region | awk -F= '{print $2}' | tr -d '"')
+sleep 10
+
+# Assign the cluster-admin role to the "client" user created thanks to the master_auth parameter (../../../../modules/distribution/gke/main.tf).
+cat <<EOF | kubectl apply -f -
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: client-binding
+subjects:
+- kind: User
+  name: client
+roleRef:
+  kind: ClusterRole
+  name: "cluster-admin"
+  apiGroup: rbac.authorization.k8s.io
+EOF
```

Comment:
Can this be accomplished with the related terraform resource? https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/cluster_role_binding_v1